docs/conf.py

```diff
@@ -11,8 +11,6 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os
```

Comment:
Rather than remove these, can you move them next to line 17 where they would be used if those lines were uncommented?